### ISSUE TITLE
Uninitialized string offset 0

### ISSUE THREAD

**Issue by dktapps** [pmmp/PocketMine-MP#4625]
Link to crashdump: https://crash.pmmp.io/view/5588970

### Additional comments
I'm not very clear why this happens - it shouldn't be possible if `SimpleCommandMap` is doing its job properly.

**Comment by dktapps** [comment]
I suppose this was never reported on PM3 due to the `Internal server error` catch-all. PM4 for the win 😎

**Comment by dktapps** [comment]
Reproduced on PM3 with the following command: `tp 0 128 \`
```
[17:31:32] [Server thread/CRITICAL]: ErrorException: "Uninitialized string offset 0" (EXCEPTION) in "pmsrc/src/pocketmine/command/defaults/VanillaCommand" at line 54
[17:31:32] [Server thread/CRITICAL]: #0 pmsrc/src/pocketmine/command/defaults/VanillaCommand(54): pocketmine\utils\Utils::errorExceptionHandler(integer 2, string[29] Uninitialized string offset 0, string[113] C:\Users\dylan-work\Documents\projects\pocketmine-mp\oldstable\src\pocketmine\co, integer 54)
[17:31:32] [Server thread/CRITICAL]: #1 pmsrc/src/pocketmine/command/defaults/TeleportCommand(114): pocketmine\command\defaults\VanillaCommand->getRelativeDouble(double 244.8702, object pocketmine\Player, string[0] )
[17:31:32] [Server thread/CRITICAL]: #2 pmsrc/src/pocketmine/command/SimpleCommandMap(248): pocketmine\command\defaults\TeleportCommand->execute(object pocketmine\Player, string[2] tp, array[3])
[17:31:32] [Server thread/CRITICAL]: #3 pmsrc/src/pocketmine/Server(1809): pocketmine\command\SimpleCommandMap->dispatch(object pocketmine\Player, string[10] tp 0 128 \)
[17:31:32] [Server thread/CRITICAL]: #4 pmsrc/src/pocketmine/Player(2352): pocketmine\Server->dispatchCommand(object pocketmine\Player, string[10] tp 0 128 \)
[17:31:32] [Server thread/CRITICAL]: #5 pmsrc/src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(239): pocketmine\Player->chat(string[11] /tp 0 128 \)
[17:31:32] [Server thread/CRITICAL]: #6 pmsrc/src/pocketmine/network/mcpe/protocol/CommandRequestPacket(54): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleCommandRequest(object pocketmine\network\mcpe\protocol\CommandRequestPacket)
[17:31:32] [Server thread/CRITICAL]: #7 pmsrc/src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(109): pocketmine\network\mcpe\protocol\CommandRequestPacket->handle(object pocketmine\network\mcpe\PlayerNetworkSessionAdapter)
[17:31:32] [Server thread/CRITICAL]: #8 pmsrc/src/pocketmine/network/mcpe/protocol/BatchPacket(133): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleDataPacket(object pocketmine\network\mcpe\protocol\CommandRequestPacket)
[17:31:32] [Server thread/CRITICAL]: #9 pmsrc/src/pocketmine/network/mcpe/PlayerNetworkSessionAdapter(109): pocketmine\network\mcpe\protocol\BatchPacket->handle(object pocketmine\network\mcpe\PlayerNetworkSessionAdapter)
[17:31:32] [Server thread/CRITICAL]: #10 pmsrc/src/pocketmine/Player(3336): pocketmine\network\mcpe\PlayerNetworkSessionAdapter->handleDataPacket(object pocketmine\network\mcpe\protocol\BatchPacket)
[17:31:32] [Server thread/CRITICAL]: #11 pmsrc/src/pocketmine/network/mcpe/RakLibInterface(169): pocketmine\Player->handleDataPacket(object pocketmine\network\mcpe\protocol\BatchPacket)
[17:31:32] [Server thread/CRITICAL]: #12 pmsrc/vendor/pocketmine/raklib/src/server/ServerHandler(95): pocketmine\network\mcpe\RakLibInterface->handleEncapsulated(string[15] 127.0.0.1 57146, object raklib\protocol\EncapsulatedPacket, integer 0)
[17:31:32] [Server thread/CRITICAL]: #13 pmsrc/src/pocketmine/network/mcpe/RakLibInterface(109): raklib\server\ServerHandler->handlePacket()
[17:31:32] [Server thread/CRITICAL]: #14 pmsrc/src/pocketmine/network/mcpe/RakLibInterface(99): pocketmine\network\mcpe\RakLibInterface->process()
[17:31:32] [Server thread/CRITICAL]: #15 pmsrc/vendor/pocketmine/snooze/src/SleeperHandler(113): pocketmine\network\mcpe\RakLibInterface->pocketmine\network\mcpe\{closure}()
[17:31:32] [Server thread/CRITICAL]: #16 pmsrc/vendor/pocketmine/snooze/src/SleeperHandler(75): pocketmine\snooze\SleeperHandler->processNotifications()
[17:31:32] [Server thread/CRITICAL]: #17 pmsrc/src/pocketmine/Server(2155): pocketmine\snooze\SleeperHandler->sleepUntil(double 1638984692.5971)
[17:31:32] [Server thread/CRITICAL]: #18 pmsrc/src/pocketmine/Server(1992): pocketmine\Server->tickProcessor()
[17:31:32] [Server thread/CRITICAL]: #19 pmsrc/src/pocketmine/Server(1586): pocketmine\Server->start()
[17:31:32] [Server thread/CRITICAL]: #20 pmsrc/src/pocketmine/PocketMine(330): pocketmine\Server->__construct(object BaseClassLoader, object pocketmine\utils\MainLogger, string[63] C:\Users\dylan-work\Documents\projects\pocketmine-mp\oldstable\, string[71] C:\Users\dylan-work\Documents\projects\pocketmine-mp\oldstable\plugins\)
[17:31:32] [Server thread/CRITICAL]: #21 pmsrc/src/pocketmine/PocketMine(360): pocketmine\server()
```

**Comment by dktapps** [comment]
Fixed by 448f26cefcd748533921605c616de5c335f823f9.
